### DISCUSSION TITLE
HW 3 part 1 Q2

### DISCUSSION THREAD

**Discussion by saewon1** [cooperrc/computational-mechanics#110] [General]
I think that there is a typo with the coefficient of restitution equation:
![image](https://user-images.githubusercontent.com/112506894/194349475-44d89982-1b9d-4921-85eb-f00be7ea5298.png)

Shouldn't vy (non prime) be the velocity before the impact and not after?

**Comment by cooperrc** [comment]
yes, sorry $v_y$ should be "velocity _before_ impact"
